src/main/java/com/exasol/validation/PlatformValidatorFactory.java

```diff
@@ -19,14 +19,15 @@ private PlatformValidatorFactory() {
      * Create a new instance of the {@link PlatformValidator} depending on the {@link PlatformName}.
      *
      * @param repositoryContent {@link GitBranchContent} with the project's content
-     * @param platform release platform details
+     * @param platform          release platform details
+     * @param validationReport  instance of {@link ValidationReport}
      * @return new instance of {@link PlatformValidator}
      */
     public static PlatformValidator createPlatformValidator(final GitBranchContent repositoryContent,
-            final Platform platform) {
+            final Platform platform, ValidationReport validationReport) {
         final PlatformName releasePlatform = platform.getPlatformName();
         if (releasePlatform == GITHUB) {
-            return new GitHubPlatformValidator(repositoryContent, (GitHubPlatform) platform);
+            return new GitHubPlatformValidator(repositoryContent, (GitHubPlatform) platform, validationReport);
         } else {
             throw new UnsupportedOperationException("E-RR-VAL-9: Validation for release platform " + releasePlatform
                     + " is not supported. Please choose one of: github");
```

Comment:
replace hardocded github by constant or list

Reply:
I have maven there, but it's not supported yet. I can replace it later